lib/active_admin/router.rb

```diff
@@ -24,7 +24,7 @@ def define_root_routes(router)
             root namespace.root_to_options.merge(to: namespace.root_to)
           else
             namespace namespace.name do
-              root namespace.root_to_options.merge(to: namespace.root_to)
+              root namespace.root_to_options.merge(to: namespace.root_to, as: :root)
```

Comment:
does this results in `root_path` or `admin_root_path`? (given `namespace.name == 'admin'`)